assets/scss/base/_table.scss

```diff
@@ -55,6 +55,10 @@ table {
   padding: 20px 12px 20px 12px;
 }
 
+.table-cell--indented {
+  padding-left: 30px;
```

Comment:
does this need to be table specific? potentially could be a generic .gutter-left class?